### PR TITLE
Add ability to spawn runners in DO vpc

### PR DESCRIPTION
**Pull request by tomjohnburton**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -57,7 +57,9 @@ steps:
   image: plugins/docker
   settings:
     auto_tag: true
-    repo: drone/autoscaler
+    repo: ghcr.io/chainflip-io/autoscaler
```
Comment (tboerger) [inline .drone.yml:60]: This should not be part of the pr.

File: .gitignore [hunk 2]
```diff
@@ -7,4 +7,5 @@ vendor
 *.bak
 *.out
 *.db
-*.env
\ No newline at end of file
+*.env
+.idea
```
Comment (tboerger) [inline .gitignore:11]: Ignores for your dev environment should not be part of the project ignore.

File: config/config.go [hunk 3]
```diff
@@ -141,6 +141,7 @@ type (
 			SSHKey       string
 			Size         string
 			Firewall     string
+			Vpc     string
```
Comment (tboerger) [inline config/config.go:144]: Wrong formatting

File: drivers/digitalocean/provider.go [hunk 4]
```diff
@@ -25,6 +25,7 @@ type provider struct {
 	size      string
 	image     string
 	firewall  string
+	vpc		  string
```
Comment (tboerger) [inline drivers/digitalocean/provider.go:28]: Also wrong formatting

**Review by tboerger — COMMENTED** [review]
I have added some comments, but beside that the vpc option have not been added to the config struct.

**Comment by tomjohnburton** [comment]
Go into a bit of a death spiral with the PR with some broken tests. Restarted with this one #86
